src/components/searchpanel/SearchPanel.module.scss

```diff
@@ -8,6 +8,8 @@
     display: flex;
```

Comment:
Not here but remove lines 1-5 since they are commented out

Reply:
Removed them